test/cmp_hdr_test.c

```diff
@@ -414,8 +416,8 @@ static int test_HDR_init_with_subject(void)
 
     fixture->expected = 1;
     if (!TEST_ptr(subject = X509_NAME_new())
-        || !TEST_true(X509_NAME_ADD(subject, "CN", "Common Name"))
-        || !TEST_true(OSSL_CMP_CTX_set1_subjectName(fixture->cmp_ctx,
+            || !TEST_true(X509_NAME_ADD(subject, "CN", "Common Name"))
+            || !TEST_true(OSSL_CMP_CTX_set1_subjectName(fixture->cmp_ctx,
```

Comment:
next line becomes unaligned with this change

Reply:
Thanks, fixed.